.github/workflows/release.yml

```diff
@@ -92,7 +99,8 @@ jobs:
       id: branch-push
       shell: bash
       env:
-        SEMANTIC_VERSION: ${{ inputs.semantic_version }}
+        # TODO(GA): specify ${{ inputs.stable_semantic_version }}
```

Comment:
The `SEMANTIC_VERSION` environment variable is used in a script `get-or-create-release-branch`. The script may create a new release branch based on that environment variable. For GA, we don't specify two environment variables for the script because it will create a stable release branch from `inputs.stable_semantic_version` and unstable crates are part of the stable release branch.